src/lib.rs

```diff
@@ -404,6 +404,9 @@ pub use serde_value;
 #[cfg(feature = "api")]
 pub use url;
 
+
+pub mod deep_merge;
```

Comment:
```suggestion
mod deep_merge;
```

The only useful thing in this mod is the trait.